engine/time/src/test/java/io/deephaven/time/TestDateTimeUtils.java

```diff
@@ -2928,6 +3091,7 @@ public void testNanosOfMilli() {
     public void testNanosOfDay() {
```

Comment:
The prior test is for `nanosOfMilli` but it doesn't have a LDT change.  That implies that the methods were missed for some of these shorter times.  Check the tests just above this to see what methods need LDT support.

Reply:
These methods also didn't have `LocalTime` support. I assumed it was intentional.

Reply:
https://github.com/deephaven/deephaven-core/issues/5589